scripts/test/hive.sh

```diff
@@ -47,5 +47,5 @@ export PYTHONPATH=$GOPATH/src/sqlflow.org/sqlflow/python
 
 go generate ./...
 go install ./...
-gotest -p 1 -covermode=count -coverprofile=coverage.txt -timeout 1200s  -v ./...
+gotest -p 1 -covermode=count -coverprofile=coverage.txt -timeout 1800s  -v ./...
```

Comment:
Why increase the timeout threshold?

Reply:
Because the `TestEnd2EndHive` unittest consumes lots of time.